src/components/singlevideo/SingleVideo.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import ReactPlayer from "react-player";
+import "./singlevideo.css";
+import { videos } from "backend/db/videos";
+import { BiLike } from "react-icons/bi";
+import { FaClock } from "react-icons/fa";
+
+const SingleVideo = ({ id }) => {
+  return videos.map((video) =>
```

Comment:
```
const video = videos.find(video=>video._id === id )

```
then show this in ui, no need to map over video

Reply:
Thanks for the suggestion jantu , will do that